README.md

```diff
@@ -10,24 +10,40 @@
 # cromshell
  A script for submitting workflows to a cromwell server and monitoring / querying their results.
 
-requires `column`, `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
+Current version: 2.0.0.beta 
 
-Current version: 2.0.0 
+Cromshell 2 is the next step in the evolution Cromshell. It offers many of the same
+functions as Cromshell 1 but has been rebuilt in python with many added benefits such as:
+- Added integration and unit tests to insure reliability 
+- Multiple way of installation (source, brew tap, and pypi)
+- Moduler code architecture that makes easy to understand so users are encouraged 
+to submit features they'd like to see in the tool.
+- Developer documentation showing how to add new features and commands to the tool.
 
 ## Examples:
 
 ```
-         cromshell-alpha submit workflow.wdl inputs.json options.json dependencies.zip
-         cromshell-alpha status
-         cromshell-alpha -t 20 metadata
-         cromshell-alpha logs -2
+         cromshell-beta submit workflow.wdl inputs.json options.json dependencies.zip
```

Comment:
Not a comment on this PR, but I still think #193 will be a big convenience for users.

Reply:
added to the features in readme